Makefile

```diff
@@ -3,11 +3,24 @@ test: venv
 	venv/bin/pre-commit install
 	venv/bin/pre-commit run --all-files
 
+.PHONY: builddeb
+builddeb: autoversion
```

Comment:
Following the [general package build pipeline](https://github.com/ocf/shared-pipeline/blob/205123a15fb74a9961d4a62f506f9b703bb275bf/vars/packagePipeline.groovy#L10), I think you'd also want to have a per distribution version of this like `package_%` (see [ocflib for inspiration](https://github.com/ocf/ocflib/blob/ae3bcade3290e921e41ab51c9ba5537c3e04efa5/Makefile#L28-L33)). Then have those output files into `dist_%`, and then the whole pipeline will handle the testing, build, and upload all automatically and the `Jenkinsfile` can just be like this or something (intentionally not including `jessie` since we should be fully moved off it in the next couple weeks now):

```
packagePipeline(
    dists: ['stretch', 'buster'],
)
```

Reply:
Yeah, this currently just spits out the package in the directory above `utils/`, which isn't great. Ideally we'd want this to be more self-contained and build in docker or something like that and then just split out the package files once done like builds in ocflib/approve/etc. do.